mlir/test/Dialect/Vector/invalid.mlir

```diff
@@ -646,6 +655,14 @@ func.func @test_vector.transfer_write(%arg0: memref<?xf32>, %arg1: vector<7xf32>
 
 // -----
 
+func.func @test_vector.transfer_write(%vec_to_write: vector<3x4xi32>, %output_memref: memref<?xindex>) {
+  %c3_idx = arith.constant 3 : index
```

Comment:
Could this be made symmetric to `@test_vector.transfer_read, i.e. use `i32` instead?